src/System.ServiceModel.Syndication/tests/BasicScenarioTests.cs

```diff
@@ -524,37 +524,6 @@ public static void SyndicationFeed_Atom_TestDisjointItems()
         }
 
         [Fact]
-        public static async Task SyndicationFeed_RSS_Optional_TextInput()
-        {
-            // *** SETUP *** \\
-            XmlReaderSettings setting = new XmlReaderSettings();
-            setting.Async = true;
-            XmlReader reader = null;
-            Task<SyndicationFeed> rss = null;
-            CancellationToken ct = new CancellationToken();
-
-            try
-            {
-                // *** EXECUTE *** \\
-                reader = XmlReader.Create(@"rssSpecExample.xml", setting);
```

Comment:
Do you need to remove rssSpecExample.xml?

Reply:
No, the file is still used by some other tests.